wsclient/src/main/java/org/hpccsystems/ws/client/HPCCFileSprayClient.java

```diff
@@ -1005,7 +1005,7 @@ public ProgressResponseWrapper sprayVariable(DelimitedDataOptions options, DropZ
 
         SprayVariable request = new SprayVariable();
         request.setSourceIP(targetDropZone.getNetAddress());
-        request.setSourcePath(Utils.ensureTrailingPathSlash(targetDropZone.getPath()) + sourceFileName);
+        request.setSourcePath(Utils.ensureTrailingPathSlash(Utils.appendPathSections(targetDropZone.getPath(),sourceFileName)));
```

Comment:
Should there be a Utils function that builds the path and ensures there is trailing path slash?